ddtrace/internal/agent.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from ddtrace.vendor import six
+from ddtrace import compat
+
+from .uds import UDSHTTPConnection
+
+
+HOSTNAME = os.environ.get("DD_AGENT_HOST", os.environ.get("DATADOG_TRACE_AGENT_HOSTNAME", "localhost"))
+PORT = int(os.environ.get("DD_AGENT_PORT", os.environ.get("DD_TRACE_AGENT_PORT", 8126)))
+URL = os.environ.get("DD_TRACE_AGENT_URL", "http://%s:%d" % (HOSTNAME, PORT))
```

Comment:
Be aware that this is not going to allow changes to the environ once the program is loaded. I don't know if it's an issue?

Reply:
This is how these variables worked before in `Tracer`. Not sure if it's the best thing, but it should probably be in a separate PR if we make these run-time look ups.